### PR TITLE
Optionally download modules from the gocenter.io

### PR DESCRIPTION
**Pull request by rimusz**

### CODE REVIEW

File: Makefile [hunk 1]
```diff
@@ -5,6 +5,12 @@ VERSION := $(shell sed -n -e 's/version:[ "]*\([^"]*\).*/\1/p' plugin.yaml)
 DIST := $(CURDIR)/_dist
 LDFLAGS := "-X main.version=${VERSION}"
 
+PHONY: modules
+modules:
+	export GOPROXY=https://gocenter.io && \
+	export GO111MODULE=on && \
+	go mod download
+
```
Comment (hickeyma) [inline Makefile]: How about writing it something like this (might need to rework the syntax):

```
MOD_PROXY_URL ?= https://gocenter.io

.PHONY: bootstrap
bootstrap: 
                export GO111MODULE=on && \
                export GOPROXY=$(MOD_PROXY_URL) && \
	        go mod download
```

Comment (rimusz) [reply Makefile]: sure

Comment (rimusz) [reply Makefile]: done

File: Makefile [hunk 2]
```diff
@@ -4,6 +4,13 @@ HELM_PLUGIN_DIR ?= $(HELM_HOME)/plugins/helm-2to3
 VERSION := $(shell sed -n -e 's/version:[ "]*\([^"]*\).*/\1/p' plugin.yaml)
 DIST := $(CURDIR)/_dist
 LDFLAGS := "-X main.version=${VERSION}"
+MOD_PROXY_URL ?= https://gocenter.io
+
+PHONY: bootstrap
+bootstrap:
+	export GO111MODULE=on && \
+    export GOPROXY=$(MOD_PROXY_URL) && \
```
Comment (hickeyma) [inline Makefile]: Do you mins aligning with other `bootstrap` commands?

Comment (rimusz) [reply Makefile]: sure, that VS Code show is differently

Comment (rimusz) [reply Makefile]: done

**Pull request merged**